quasar-cli-vue3-webpack-javascript/e2e/CheckAppExposes.cy.ts

```diff
@@ -0,0 +1,302 @@
+        AppButtonName: Constants.commonPhrases.button,
+        appButtonDiv: Constants.elementsText.appButtonDiv,
+        clickMeButton: Constants.elementsText.appButtonClickMeButton,
+        appListName: Constants.elementsText.appExposesList,
+        appListDiv: Constants.elementsText.appListDiv,
+        host: 3001
+    },
+
+    {
+        appExposesName: Constants.elementsText.appExposesName,
+        appGeneralName: Constants.elementsText.appGeneralName,
+        appGeneralBanner: Constants.elementsText.appGeneralBanner,
+        appExposesComponentsButton: Constants.elementsText.appExposesComponentsButton,
+        appGeneralRouteButton: Constants.elementsText.appGeneralRouteButton,
+        appGeneralSubheader1: Constants.elementsText.appGeneralSubheader1,
+        appGeneralSubheader2: Constants.elementsText.appGeneralSubheader2,
+        appGeneralCounter: Constants.elementsText.appGeneralCounter,
+        clickMeButton: Constants.elementsText.appButtonClickMeButton,
+        appListDiv: Constants.elementsText.appListDiv,
+        appGeneralName1: Constants.elementsText.appGeneralName1,
+        appGeneralName2: Constants.elementsText.appGeneralName2,
+        appGeneralName3: Constants.elementsText.appGeneralName3,
+        appGeneralName4: Constants.elementsText.appGeneralName4,
+        host: 3002
+    }
+]
+
+appsData.forEach((
+    property: {
+        appExposesName?: string,
+        appExposesBanner?: string,
+        appGeneralBanner?: string,
+        appExposesComponentsButton?: string,
+        AppButtonName?: string,
+        appButtonDiv?: string,
+        clickMeButton?: string,
+        appListName?: string,
+        appListDiv?: string,
+        appListItems?: object,
+        appGeneralName?: string,
+        appGeneralRouteButton?: string,
+        appGeneralSubheader1?: string,
+        appGeneralSubheader2?: string,
+        appGeneralCounter?: string,
+        appGeneralName1?: string,
+        appGeneralName2?: string,
+        appGeneralName3?: string,
+        appGeneralName4?: string,
+        host: number
+        
+    }
+) => {
+    describe(`Check App Exposes and App General starts and running`, () => {
+        beforeEach(() => {
+            basePage.openLocalhost(property.host)
+        })
+        it (`Check App Exposes elements`, () => {
+            basePage.skipTestByCondition(property.host === 3002)
+            basePage.clickElementWithText({
+                selector: baseSelectors.divElement,
+                text: String(property.appExposesName)
+            })
+            basePage.checkElementWithTextPresence({
+                selector: baseSelectors.h4,
+                text: String(property.appExposesBanner)
+            })
+            basePage.clickElementBySelector({
+                selector: baseSelectors.button
+            })
+            basePage.clickElementBySelector({
+                selector: baseSelectors.button
+            })
```

Comment:
Why do you click on button for the second time?

Reply:
to open left side menu

Reply:
<img width="310" alt="Screenshot 2023-01-05 at 3 10 58 PM" src="https://user-images.githubusercontent.com/78354418/210766841-01bcddcc-9019-438a-90b0-e8843b05f08a.png">
<img width="345" alt="Screenshot 2023-01-05 at 3 11 05 PM" src="https://user-images.githubusercontent.com/78354418/210766868-e20fef6e-6512-41c2-be60-e9ad87ed84a1.png">